src/js/ripple/orderbook.js

```diff
@@ -8,16 +8,22 @@
 //  - trade
 //  - transaction
 
-var _            = require('lodash');
-var util         = require('util');
-var extend       = require('extend');
-var assert       = require('assert');
-var async        = require('async');
+'use strict';
+
+var _ = require('lodash');
+var util = require('util');
+var extend = require('extend');
+var assert = require('assert');
+var async = require('async');
 var EventEmitter = require('events').EventEmitter;
-var Amount       = require('./amount').Amount;
-var UInt160      = require('./uint160').UInt160;
-var Currency     = require('./currency').Currency;
-var log          = require('./log').internal.sub('orderbook');
+var Amount = require('./amount').Amount;
+var UInt160 = require('./uint160').UInt160;
+var Currency = require('./currency').Currency;
+var log = require('./log').internal.sub('orderbook');
+
+function assertValidNumber(number, message) {
+  assert(!_.isNull(number) && !isNaN(number), message);
```

Comment:
Maybe we should move `assert-diff` to dependencies and use `assert.number` https://github.com/mcavage/node-assert-plus/blob/master/assert.js#L126-L137.